src/utils/numbers/convertEth.js

```diff
@@ -10,27 +10,31 @@ const units = Object.keys(rawUnits).reduce((res, unit) => ({
 }), {})
 
 function convertEth(value: string, from: string, to: string): ?string {
```

Comment:
we really don't need this helper
we need only `toGwei` / `fromGwei` helpers and not in `utils/numbers` scope